api_tests/test_lib.js

```diff
@@ -22,10 +22,14 @@ module.exports.logger = function () {
     return logger;
 };
 
-module.exports.sleep = time => {
+async function sleep (time) {
     return new Promise((res, rej) => {
         setTimeout(res, time);
     });
+}
+
+module.exports.sleep = time => {
```

Comment:
why not just set this `= sleep` ?

Reply:
I did not know it was an option :)

Reply:
I fixed that locally and will push with a new PR.